kornia/geometry/linalg.py

```diff
@@ -200,5 +197,31 @@ def transform_points(trans_01: torch.Tensor, points_1: torch.Tensor) -> torch.Te
     points_0 = points_0.reshape(shape_inp)
     return points_0
 
+
+def points_lines_distances(pts: torch.Tensor, lines: torch.Tensor):
```

Comment:
```suggestion
def point_line_distance(point: Tensor, line: Tensor) -> Tensor:
```
I'd use as singular which is at least the minimum supported and same convention as other functions
example here: https://github.com/kornia/kornia/blob/master/kornia/utils/draw.py#L26